src/main/java/org/prebid/server/spring/config/ServiceConfiguration.java

```diff
@@ -56,12 +56,14 @@ CacheService cacheService(
             @Value("${cache.scheme}") String scheme,
             @Value("${cache.host}") String host,
             @Value("${cache.query}") String query,
+            @Value("${cache.hostPath:}") String hostPath,
```

Comment:
Qs:
- should it be configurable in this way but not the flag like `auction.include-cache-host-path` ?
- should it be a part of `CacheService` ? m.b. the `ExchangeService.java` is better way.

Reply:
cant find auction.include-cache-host-path
i think its better to stay it here, because it belongs to cache behaviour